### PR TITLE
#174 added Administrator role

### PR DESCRIPTION
**Pull request by krzyk**
#174 
* added administrator role privileges
* added test to verify them

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#277](https://github.com/timelesslounge/timelessis/issues/277) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#277](https://github.com/timelesslounge/timelessis/issues/277) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/a9ab8ff0-2c4a-4564-9e46-72d5cd7504dc); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/access_control/administrator_privileges.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from timeless.access_control.methods import Method
```
Comment (vryazanov) [inline timeless/access_control/administrator_privileges.py]: @krzyk unused import, let's remove it.

**Comment by krzyk** [comment]
@vryazanov updated

**Comment by vryazanov** [comment]
@rultor  merge

**Comment by emilianodellacasa** [comment]
@krzyk there is a problem in a puzzle, please check

**Comment by krzyk** [comment]
@emilianodellacasa fixed

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 17 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#277](https://github.com/timelesslounge/timelessis/issues/277) is not in the agenda of @vryazanov[/z](https://www.0crat.com/u/vryazanov), can't inspect

**Comment by 0crat** [comment]
The job [#277](https://github.com/timelesslounge/timelessis/issues/277) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by vryazanov** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/277#issuecomment-464996201))

@vryazanov This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#277](https://github.com/timelesslounge/timelessis/issues/277) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 13hr
  * @vryazanov[/z](https://www.0crat.com/u/vryazanov) will receive 20 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23277%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@vryazanov According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@vryazanov thanks

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request merged**